build

```diff
@@ -58,10 +58,84 @@ create_aliases() {
   done <"$ALIASES"
 }
 
+create_hyprcursors() {
+  local src_dir="$1"
+  local out_dir="$2"
+  local theme_name="$3"
+  local hypr_dir="$out_dir/hyprcursors"
+  local filename dir cursor_dir striped base_name symlink target cursor_dir theme_comment
+
+  [ -d "$src_dir" ] || return 1
+  [ -d "$out_dir" ] || mkdir -p "$out_dir"
+  [ -d "$hypr_dir" ] || mkdir -p "$hypr_dir"
+
+  for file in "$src_dir"/*.svg; do
+    filename=$(basename -- "$file")
+
+    if [[ "$filename" == *_24.svg ]]; then
+      continue
+    fi
+    if [[ "$filename" == *-[0-9][0-9].svg ]]; then
+      dir="${filename%-[0-9][0-9].svg}"
+      cursor_dir="$hypr_dir/$dir"
+      striped="${filename#"$dir"}"
+
+      [ -d "$cursor_dir" ] || mkdir -p "$cursor_dir"
+      cp -- "$file" "$cursor_dir"
+    else
+      dir="${filename%.svg}"
+      cursor_dir="$hypr_dir/$dir"
+
+      [ -d "$cursor_dir" ] || mkdir -p "$cursor_dir"
+
+      cp -- "$file" "$cursor_dir"
+    fi
+  done
+
+  for config in "$CONFIG_DIR"/*.hl; do
+    [ -f "$config" ] || continue
+
+    base_name="$(basename "$config" .hl)"
+    cursor_dir="$hypr_dir/$base_name"
+    [ -d "$cursor_dir" ] || continue
+    cp "$config" "$cursor_dir/meta.hl"
+  done
+
+  while read -r symlink target; do
+    cursor_dir="$out_dir/$target"
+    if [ -f "$cursor_dir"/meta.hl ]; then
+      echo "define_override = $symlink" >>"$cursor_dir"/meta.hl
+    fi
+  done <"$ALIASES"
+
+  theme_comment="$(grep Comment "$src_dir"/index.theme)"
+  cat >"$out_dir"/manifest.hl <<-EOM
+		name = ${theme_name%-cursors}
+		description = ${theme_comment#Comment=}
+		version = $(git show -s --format=%cd --date=format:%Y%m%d HEAD)
+		cursors_directory = hyprcursors
+	EOM
+}
+
+compile_hyprcursors() {
+  local src_dir="$1"
+  local out_dir="$2"
+  local theme_name="$(basename "$1")"
+  local compile_dir="$1/../theme_${theme_name%-cursors}"
+  local filename dir cursor_dir striped base_name symlink target cursor_dir
+
+  [ -d "$src_dir" ] || return 1
+  [ -d "$out_dir" ] || mkdir -p "$out_dir"
+
+  hyprcursor-util -c "$src_dir" >/dev/null
+  cp -a "$compile_dir/." "$out_dir/"
+}
+
 SCRIPT_DIR="$(dirname "$0")"
 SRC_DIR="$SCRIPT_DIR/src"
 OUT_DIR="$SCRIPT_DIR/dist"
 PNG_DIR="$SCRIPT_DIR/pngs"
+HL_DIR="$SCRIPT_DIR/hl"
 BUILD_DIR="$SCRIPT_DIR/build"
```

Comment:
I meant to remove this in my refactoring, would be good to remove it :+1: 
```suggestion
```

Reply:
Removed, and cleaned up the unused variables in the in the loops below.